Analytics/App_Plugins/Analytics/backOffice/AnalyticsTree/controllers/Settings.Controller.js

```diff
@@ -106,8 +106,12 @@
                 //Don't need anything from response.data back
 
                 //Display Success message
-                notificationsService.success("Success profile details have been saved");
+                notificationsService.success(localizationService.localize("analytics_profileDetailsSaved"));
             });
+
+            //Sync ('refresh') the tree!
+            //This should work, but somehow it seems to fails sometimes when you click "save" and receive an error in a dialog on the right.
+            //navigationService.syncTree({ tree: 'analyticsTree', path: [-1, -1], forceReload: false, activate: true });
```

Comment:
What is the reason for adding the sync tree stuff @abjerner ? 
As I thought the PR was for translations?